editor/test/editor/pipeline_test.clj

```diff
@@ -201,3 +201,73 @@
                   :default-animation "gurka"
                   :material (-> material-target :resource resource/proj-path)}
                 (select-keys pb-data [:tile-set :default-animation :material]))))))))
+
+(defrecord TestResource [proj-path]
+  resource/Resource
+  (children [_] [])
+  (ext [_] (FilenameUtils/getExtension proj-path))
+  (resource-type [this] {:build-ext (str (resource/ext this) "c")})
+  (source-type [_] :file)
+  (exists? [_] true)
+  (read-only? [_] true)
+  (path [_] (subs proj-path 1))
+  (abs-path [_] proj-path)
+  (proj-path [_] proj-path)
+  (resource-name [_] (FilenameUtils/getName proj-path))
+  (workspace [_])
+  (resource-hash [_] (hash proj-path))
+  (openable? [_] false)
+  (editable? [_] false))
+
+(defn- build-resource [proj-path]
+  (workspace/make-build-resource (->TestResource proj-path)))
```

Comment:
Would `test-util/make-fake-file-resource` work for your use case?

Reply:
I kinda like this one more, just a project path basically.